PAS_Database/dbo/Stored Procedures/Procs1/UpdateWorkOrderQuoteTable.sql

```diff
@@ -98,8 +99,8 @@ BEGIN
 							 WOQC.TaskName = T.Description,
 							 WOQC.BillingName = (case when WOQC.BillingMethodId = 1 then 'T&M'  when  WOQC.BillingMethodId = 2 then 'Actual' else '' End),
 							 WOQC.MarkUp = p.PercentValue,
-							 WOQC.ChargeType = c.ChargeType,
-							 WOQC.GlAccountName=gl.AccountName
+							 WOQC.ChargeType = c.ChargeType
+							 --WOQC.GlAccountName=gl.AccountName
```

Comment:
No need to comment code, work as expected early. handle for get & display time with "As Per GL Allocation"